arm-servicefabric/2016-09-01/examples/ClusterPutOperation_exmaple.json

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
It means - we are looking for two examples for PUT here. (1) One is the minimum set of parameters to be specified in order to PUT a cluster and (2) one with all the parameters that can be specified while PUTing cluster. Your provided example belongs to (2). Please check if you can provide example for (1) as well.